resources/results.py

```diff
@@ -13,13 +15,23 @@
 )
 
 path_to_gnomad_rf = (
-    "gs://gcp-public-data--gnomad/release/3.1/pca/gnomad.v3.1.RF_fit.pkl"
+    # "gs://gcp-public-data--gnomad/release/3.1/pca/gnomad.v3.1.RF_fit.pkl"
+    "gs://ccdg/wlu/pca/gnomad_v3.1_pop.ccdg_ukb_gnomad_variants.RF_fit.pickle"
 )
 path_to_gnomad_loadings = (
-    "gs://gcp-public-data--gnomad/release/3.1/pca/gnomad.v3.1.pca_loadings.ht"
+    # "gs://gcp-public-data--gnomad/release/3.1/pca/gnomad.v3.1.pca_loadings.ht" # public: 76,399
+    # "gs://gnomad/sample_qc/ht/genomes_v3.1/gnomad_v3.1_pca_loadings.ccdg_ukb_gnomad_variants.ht" # new: 224,591
+    "gs://ccdg/wlu/pca/genomes_v3.1/gnomad_v3.1_pca_loadings.ccdg_ukb_gnomad_variants.ht"
```

Comment:
Here I am using the latest gnomAD PCA results from the selected variant set. but those do not seem to be public, so I copied them to the `ccdg` bucket.

Reply:
Makes sense, can you add that as a comment in the code?